modules/Bio/EnsEMBL/Analysis/Hive/Config/StarScallopRnaseq.pm

```diff
@@ -0,0 +1,827 @@
+        commandline_params => $self->o('blast_type') eq 'wu' ? '-cpus=' . $self->o('use_threads') . ' -hitdist=40' : '-num_threads ' . $self->o('use_threads') . ' -window_size 40',
+      },
+      -rc_name => 'blast10GB',
+    },
+
+    {
+      -logic_name => 'star2introns',
+      -module     => 'Bio::EnsEMBL::Analysis::Hive::RunnableDB::HiveStar2Introns',
+      -parameters => {
+        star_junctions_dir => $self->o('output_dir'),
+        intron_db          => $self->o('scallop_blast_db'),
+        source_db          => $self->o('dna_db'),
+        sample_column      => 'SM',
+        sample_id_column   => 'ID',
+        species            => $self->o('species_name'),
+      },
+      -rc_name   => 'default',
+      -flow_into => {
+        '1' => ['copy_rnaseq_star_blast_db'],
+      },
+    },
+
+    {
+      -logic_name => 'copy_rnaseq_star_blast_db',
+      -module     => 'Bio::EnsEMBL::Analysis::Hive::RunnableDB::HiveCreateDatabase',
+      -parameters => {
+        source_db   => $self->o('scallop_blast_db'),
+        target_db   => $self->o('star_rnaseq_for_layer_db'),
+        create_type => 'copy',
+        force_drop  => 1,
+      },
+      -rc_name   => 'default',
+      -flow_into => {
+        '1' => ['update_rnaseq_for_layer_biotypes_star'],
+      },
+    },
+
+    {
+      -logic_name => 'update_rnaseq_for_layer_biotypes_star',
+      -module     => 'Bio::EnsEMBL::Hive::RunnableDB::SqlCmd',
+      -parameters => {
+        db_conn => $self->o('star_rnaseq_for_layer_db'),
+        sql     => [
+          'UPDATE gene SET biotype = "rnaseq_merged" WHERE source IN ("merged")',
+          'UPDATE gene SET biotype = "rnaseq_tissue" WHERE biotype != "rnaseq_merged"',
+          'UPDATE transcript JOIN gene USING(gene_id) SET transcript.biotype = gene.biotype',
+        ],
+      },
+      -rc_name   => 'default',
+      -flow_into => {
+        '1' => ['classify_rnaseq_for_layer_models_star'],
+      },
+    },
+
+    {
+      -logic_name => 'classify_rnaseq_for_layer_models_star',
+      -module     => 'Bio::EnsEMBL::Analysis::Hive::RunnableDB::HiveClassifyTranscriptSupport',
+      -parameters => {
+        update_gene_biotype => 1,
+        classification_type => 'standard',
+        target_db           => $self->o('star_rnaseq_for_layer_db'),
+      },
+      -rc_name   => 'default',
+      -flow_into => {
+        1 => ['rnaseq_for_layer_sanity_checks_star'],
+      },
+
+    },
+
+    {
+      -logic_name => 'rnaseq_for_layer_sanity_checks_star',
+      -module     => 'Bio::EnsEMBL::Analysis::Hive::RunnableDB::HiveAnalysisSanityCheck',
+      -parameters => {
+        target_db                  => $self->o('star_rnaseq_for_layer_db'),
+        sanity_check_type          => 'gene_db_checks',
+        min_allowed_feature_counts => get_analysis_settings( 'Bio::EnsEMBL::Analysis::Hive::Config::SanityChecksStatic',
+          'gene_db_checks' )->{ $self->o('uniprot_set') }->{'rnaseq_blast'},
+      },
+
```

Comment:
Could you please remove this blank line?

Reply:
Removed.